docs/reference/supervisor.md

```diff
@@ -4012,6 +4024,206 @@ rootChildrenField.importMFNodeFromString(4, 'DEF MY_ROBOT Robot { controller "my
 
 ---
 
+#### `wb_supervisor_proto_get_type_name`
+#### `wb_supervisor_proto_is_derived`
+#### `wb_supervisor_proto_get_parent`
+
+%tab-component "language"
+
+%tab "C"
+
+```c
+#include <webots/supervisor.h>
+
+const char *wb_supervisor_proto_get_type_name(WbNodeRef proto);
```

Comment:
I believe there is an error here and at several other places in the documentation:
```suggestion
const char *wb_supervisor_proto_get_type_name(WbProtoRef proto);
```

Reply:
Fixed! I also added the missing documentation for `wb_supervisor_node_get_proto`.